pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
+
+If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.
 
 ## Issues Addressed:
 <!-- If your fix has a relating issue, link it below -->
 - Closes 
 
 ## SOURCE:
 <!-- If you can, include a source that can strengthen your claim -->
+- [ ] The changes promoted by this pull request have been validated through live research (checked on live servers)
+- [ ] The changes promoted by this pull request have been validated through sniffs (remember to share them with the opensource community!)
+- [ ] The changes promoted by this pull request come from a partially or entirelly from another project (cherry-pick). **Cherry-picks must be commited using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
 
 ## Tests Performed:
 <!-- Does it build without errors? Did you test in-game? What did you test? On which OS did you test? Describe any other tests performed -->
-- 
-- 
+- [ ] This pull request has been tested in-game by the author.
+- [ ] This pull request has been tested by the community.
```

Comment:
```suggestion
- [ ] This pull request has been tested in a production environment or independent test server before (e.g Chromiecraft).
```

Reply:
`Has been tested by someone else other than the author`?

also `independent test server` and giving CC as an example I don't think work. Several times when someone has asked to put a larger refactor or similar on CC to test the answer usually is no.

Reply:
Okay, it was just an example, but we can remove the example 🤣 
We can apply PRs to the CC PTR on request, surely not *any* and every PR as that would be overwhelming but on a per case analysis we can do

Reply:
I just remember one of Winifis refactors being turned down, that had to be tested on larger scale before it could be merged 😅